package.json

```diff
@@ -13,7 +13,8 @@
     "checks": "pnpm check:prettier && pnpm check:lint",
     "check:lint": "eslint --max-warnings=0 .",
     "check:prettier": "prettier --check .",
-    "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +"
+    "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
+    "lerna:publish": "lerna publish from-package"
```

Comment:
Are we not interested in publishing with git tags?

Reply:
For now, without a CI setup with just 3 packages, `from-package` seems like a fine approach vs  `from-git`: especially since there might be a few necessary fixes to make at the beginning, in which `from-package` is easier to manage. 

Also, having git tags for package versions might be slightly more confusing since we already have "versions" for the open payments API specs in this repo